docs/26_export_manual.md

```diff
@@ -0,0 +1,269 @@
+
+The `README.rst` file gives instructions for installing the python package, and describes various run-time options. For a catalog of the exports produced by the command, see below.
+
+
+## Database Exports
+The `rla_export` command exports many of the files necessary for independent verification of 
+the RLA, whether by candidates, parties, other organizations. 
+Specific exports are detailed below.
+
+
+
+### Minimum Data Required to Allow Public to Reproduce Audit Calculations
+
+To allow independent verification of the RLA, 
+the Colorado Department of State must provide to the public all the export files listed in this section,  along with:
+
+- all CVR files
+- all ballot manifest files
+- opportunity to observe the random seed selection
+- opportunity to observe the activities of the County Audit Boards
+- announced tabulated results for contests selected for audit
+- the risk limit
+- the error inflation factor
+
+Providing the above to the public, along with the database exports listed below will fulfill the minimum
+requirements of a publicly-verifiable audit.
+
+The data files in this section are generated based on ``sql`` query files.
+These are always produced in two formats: json and csv.
+The basename of each resulting file is the same as the basename of the query file.
+Thus, given the query file ``seed.sql``, the files ``seed.json`` and ``seed.csv``
+will be produced.
+
+#### m_selected_contest_audit_details_by_cvr.sql
+
+ For each contest under audit, and for each cast vote record presented to the 
+  Audit Board for verification that contains the given contest, 
+  the RLA system's record of the Audit Board's review of 
+  the physical ballot for that contest
+  
+
+ Field | Type | _______________Meaning_______________
+--- | --- | ---
+county_name | Text String | Name of County
+contest_name | Text String | Name of contest  
+random_sequence_index | Integer | Index in the random sequence (starting with 1)
+ imprinted_id | Text String | combination of scanner, batch and record ids  that uniquely identifies the ballot card  and may be imprinted on the card
+ ballot_type | Text String | BallotType from Dominion CVR export file, a code for the set of contests that  should be present on the physical ballot card
+ choice_per_voting_computer | List of Text Strings | List of voter choices in the given contest on the given ballot card, as interpreted by the vote-tabulation computer system (note: overvotes recorded as blank votes)
+ choice_per_audit_board | List of Text Strings | List of voter choices in the given contest on the given ballot card, as interpreted by the Audit Board (note: overvotes recorded as a too-long list of choices)
+ did_audit_board_agree | YES/NO | "YES" if the Audit Board came to consensus on the interpretation of the given ballot card; "NO" if not;  blank if the card has not been reviewed by the Audit Board.
```

Comment:
So long as the column name from the database makes sense, I'd favor using them, so that this documentation fits in well with the database documentation, formal spec, etc. There are some cases where we need to distinguish names, etc. but here, `consensus` is pretty clear and avoids confusion and the need for further documentation.